### DISCUSSION TITLE
IP address vs. DNS

### DISCUSSION THREAD

**Discussion by what-a** [iptv-org/community#164] [Q&A] [answered]
Which is better? To submit a stream URL as an IP address or as a DNS? I'm guessing DNS because it is more versatile, but I'm just guessing. Please enlighten me. 🤪

**Comment by BellezaEmporium** [comment, marked as answer]
Domain name because if the server changes IP, the DNS will automatically redirect to the correct address. Imagine not knowing the domain name of a website until IP gets changed.

**Reply by what-a** [reply]
Thank you. That makes sense. In general DNS should be used.
